PostHog/Classes/PHGPostHogConfiguration.h

```diff
@@ -123,6 +123,13 @@ typedef NSMutableURLRequest *_Nonnull (^PHGRequestFactory)(NSURL *_Nonnull);
  */
 @property (nonatomic, assign) BOOL shouldSendDeviceID;
 
+/**
+ * Whether the posthog client should load feature flags when initialized.
+ * Changing the value of this property after initializing the client will have no effect.
+ * The default value is `YES`.
+ */
+@property (nonatomic, assign) BOOL loadFeatureFlagsOnStart;
```

Comment:
```suggestion
@property (nonatomic, assign) BOOL preloadFeatureFlags;
```

Let's call it `preloadFeatureFlags` as the other SDKs.

Reply:
Updated.